commands/action.go

```diff
@@ -40,7 +40,7 @@ import (
 
 const (
 	MEMORY_LIMIT      = 256
-	TIMEOUT_LIMIT     = 60000
+	TIMEOUT_LIMIT     = 600000
```

Comment:
slipped in?

Reply:
was playing with a month or so ago...  likely on my master... good catch